src/Test_Rewriting/PriorityQueue/PriorityQueue.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace MyQueue;
+
+/// <summary>
+/// Class that implements Priority Queue based on a tuple.
+/// </summary>
+/// <typeparam name="T">Type of elements in queue.</typeparam>
+public class PriorityQueue<T>
+{
+    /// <summary>
+    /// The tuple on which queue is based.
+    /// </summary>
+    private List<(T, int)> queue = new List<(T, int)> ();

```

Comment:
```suggestion
    private List<(T, int)> queue = new();
```
Это то же самое, но короче